Rakefile

```diff
@@ -103,3 +103,6 @@ namespace :release do
     end
   end
 end
+
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new
```

Comment:
This isn't needed right?

Reply:
Not needed, we can run `rubocop` command instead of `rake rubocop`.